packages/expo/src/Updates/Updates.ts

```diff
@@ -26,7 +44,7 @@ export async function checkForUpdateAsync(): Promise<Object> {
   return returnObj;
 }
 
-export async function fetchUpdateAsync({ eventListener }: any = {}): Promise<Object> {
+export async function fetchUpdateAsync({ eventListener }: {eventListener?:UpdateEventListener} = {}) {
```

Comment:
Add back return type using the definition mentioned in the review comment below.